recidiviz/ingest/vendors/iml/iml_scraper.py

```
@@ -0,0 +1,349 @@
+            params: dict of parameters passed from the last scrape session.
+
+        Returns:
+            A list of params containing endpoint and task_type at minimum.
+        """
+        task_type = params.get('task_type', self.get_initial_task_type())
+        params_list = []
+
+        # Extract the total number of people and the cookie if we need to.
+        if self.is_initial_task(task_type):
+            num_people = self._get_num_people(content)
+            params['num_people'] = num_people
+            cookie = self._cookie
+            self._cookie = None
+        else:
+            num_people = params['num_people']
+            cookie = params['post_data']['cookie']
+
+        # Add the tasks for each person on this page.
+        params_list.extend(self._get_person_page_params(content))
+
+        # Add the next search page, if there is one.
+        next_page_params = self._get_next_search_page_params(content, params)
+        if next_page_params:
+            params_list.append(next_page_params)
+
+        # Add the cookie and the total number of people to each task.
+        for next_params in params_list:
+            next_params['num_people'] = num_people
+            next_params['post_data']['cookie'] = cookie
+
+        return params_list
+
+    def populate_data(self, content, params, ingest_info):
+        """
+        Populates the ingest info object from the content and params given
+
+        Args:
+            content: An lxml html tree.
+            params: dict of parameters passed from the last scrape session.
+            ingest_info: The IngestInfo object to populate
+        """
+        data_extractor = HtmlDataExtractor(self.yaml_file)
+        ingest_info = data_extractor.extract_and_populate_data(content,
+                                                               ingest_info)
+        if len(ingest_info.person) != 1:
+            logging.error("Data extractor didn't find exactly one person as "
```

In other cases of error-level log statements in this PR, we short-circuit. But here we silently ignore the additional persons. Should we be ignoring them instead of short-circuiting? Or, at least, should we include in the log statement that we'll ignore the extra persons?

---

Yup, I just missed the return. Fixed.